### ISSUE TITLE
Feature Consideration: Group Chat 

### ISSUE THREAD

**Issue by kekavc24** [2002Bishwajeet/no_signal#7]
Hello Biswa. Forked this repo to help you migrate to v0.15.x and noticed the approach for creating personal chats. Would you consider creating adding group chat to this project using databases. Where you have 2 databases:
* For users 
* For groups

Just like personal chats, for every group created you preset the values and have a collection that has a record of the groups the user is part of.

**Comment by 2002Bishwajeet** [comment]
Sounds Good. Would you like to give it a try?

I am kinda occupied right now so there maybe a delay in working on this feature

**Comment by kekavc24** [comment]
Yes. I'll give it a try. Lemme first fix the issues you raised on the PR 😅👍🏾

**Comment by kekavc24** [comment]
I would love your input. Should I include it in the function already there or should I create a new file and put the function there? For creating group chat database that is.

**Comment by 2002Bishwajeet** [comment]
Let's put those functions in the same file. I believe since creating a single chat and a group chat is just same so no need to write it again in the different file.
